README.md

```diff
@@ -1,21 +1,27 @@
 # OpenIDConnect
 
-Client library for consuming and working with OpenID Connect Providers
+Firezone's fork of [DockYard's client library](https://github.com/dockyard/openid_connect) for consuming and working with OpenID Connect Providers.
```

Comment:
Please revert the changes in the README that refer to firezone's fork. I'm happy to add firezone as a collaborator in the README towards the bottom